test_manager/features/steps/tck_step_implementations.py

```diff
@@ -356,6 +368,23 @@ def set_key_to_ue_uri(context: Context, key: str, ue: str):
         context.json_dict[key] = context.ue_tracker[int(ue_number) - 1][2]
 
 
+@given('sets to entity URI of "{ue}" with updated "{sub_key}" to "{sub_val}"')
+@when('sets to entity URI of "{ue}" with updated "{sub_key}" to "{sub_val}"')
+def set_to_ue_uri(context: Context, ue: str, sub_key: str, sub_val: str):
+    ue_number = ue.replace("uE", "")
```

Comment:
did you mean "ue" and not "uE"?

Reply:
no uE as feature file input to this string variable is uE1.  I just need 1.